extensions/wikia/Chat2/ChatAjax.class.php

```diff
@@ -262,6 +262,8 @@ static public function giveChatMod() {
 	private static function authenticateServer() {
 		global $wgRequest;
 
+		\Wikia\Security\CSRFDetector::markRequestAsSecure( __METHOD__ );
```

Comment:
`CSRFDetector` checks two conditions:

* the token is checked
* the HTTP method is checked

`markRequestAsSecure` marks only the second condition as true - the token needs to be checked to avoid reporting CSRF error.